app/frontend/src/locales/es/translation.json

```diff
@@ -61,7 +61,9 @@
         "showSupportingContent": "Mostrar contenido de soporte",
         "speakAnswer": "Hablar respuesta",
         "info": "Información",
-        "save": "Guardar"
+        "save": "Guardar",
+        "copy": "Copiar respuesta",
```

Comment:
Do any of these strings require human review? (i.e. they were AI generated and you're not a fluent speaker in the language?)

Reply:
Yes, all of these were AI-generated. I can personally confirm that the Dutch and English translations are accurate. So far, I’ve had very positive experiences with LLM-based translations.

Reply:
Okay, we've had some corrections needed on past translations, so I'll ask for reviewers to check the other ones.

Reply:
This reads correct to me